src/test/java/CodingPracticesTest.java

```diff
@@ -0,0 +1,63 @@
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import utilities.FileUtility;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class CodingPracticesTest {
+
+    private static final Logger logger = LogManager.getLogger(CodingPracticesTest.class);
+    private static String basePathString = "src";
+    private static final List<Path> javaFilesPath = getJavaFilesPath();
+
+    /**
+     * Get all Java Files in basePathString.
+     * @return the list of paths to the Java Files.
+     */
+    private static List<Path> getJavaFilesPath(){
+        Path basePath = Paths.get(basePathString).toAbsolutePath();
+        try {
+            return Files.walk(basePath)
+                    .filter(path -> (path.endsWith(".java") && Files.isRegularFile(path)))
+                    .collect(Collectors.toList());
+        }catch (IOException e) {
+            logger.error(e);
+            throw new RuntimeException(e);
+        }
+    }
+
+    /**
+     * Test that all Java files in the project don't have print statements.
+     */
+    @Test
+    void testJavaFilesForPrintStatements(){
+        javaFilesPath
+                .forEach(javaFilePath -> {
+                    try {
+                        byte[] encoded = Files.readAllBytes(javaFilePath);
+                        String javaFileContent = new String(encoded, StandardCharsets.US_ASCII);
+                        Pattern pattern = Pattern.compile("(System\\.(out|err)\\.print|e\\.printStackTrace\\(\\))");
```

Comment:
Please store the regular expression in a property file.

Reply:
done